.gitignore

```diff
@@ -1,25 +1,65 @@
+# Dependencies
+/node_modules
+/.pnp
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/versions
```

Comment:
Does this repo need these javascript-specific ignore paths? AFAICT the repo doesn't contain any javascript stuff. Will this change in the future? Like are we going to add an example frontend in here? 

In either case, if they're not relevant (yet), maybe let's remove these from the gitignore (at least for now)